### ISSUE TITLE
Error when parsing function value without a comma

### ISSUE THREAD

**Issue by every-time** [Nykakin/chompjs#58]
```py
src = '''
	{
		a: my_func
	}
'''

print(chompjs.parse_js_object(src))
```

Results in: `json.decoder.JSONDecodeError: Invalid control character`

Whereas each of the following works and results in: `{'a': 'my_func'}`

```py
src = '''
	{
		a: my_func,
	}
'''

print(chompjs.parse_js_object(src))
```

Note the trailing comma in the example above.

```py
src = '''
	{
		a: my_func
	}
'''

print(chompjs.parse_js_object(src.replace('\t', '').replace('\n', '')))
```

```py
src = '''
	{
		a: my_func}
'''

print(chompjs.parse_js_object(src))
```

There are various combinations to try. I think from my brief testing, a tab or a newline after a function value without a comma results in an error. In other words, a function value either has to be immediately followed by a comma, space, or a closing bracket.

**Comment by Nykakin** [comment]
Thanks for reporting!

Care to stress-test the fix in #59 before I merge and release it?

**Comment by every-time** [comment]
I'm unable to thoroughly test it now but it looks good at a glance.

I appreciate the super quick response!
